Jenkinsfile

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env groovy
 
 /* `buildPlugin` step provided by: https://github.com/jenkins-infra/pipeline-library */
-buildPlugin(useAci: true)
+buildPlugin(useContainerAgent: true, configurations: [
+  [ platform: "linux", jdk: "11" ]
```

Comment:
Are you deliberately omitting Windows builds?

Reply:
no, I forgot it was there by default